database/migration/000003_migration_ketiga.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS tentanganak;
```

Comment:
raj make it generic, jangan pake nama `tentanganak`

Reply:
done, please check new code